src/StackAdmin/Resources/Commands.ResourceManager/Cmdlets/Components/ApiVersionHelper.cs

```diff
@@ -146,11 +145,10 @@ private class ApiVersionCache
 
             static ApiVersionCache()
             {
-// TODO: Remove IfDef
-#if NETSTANDARD
-                _cache = new MemoryCache(new MemoryCacheOptions());
-#else
+#if !NETSTANDARD
```

Comment:
We can remove these - these are never compiled as NETSTANDARD

Reply:
fixed them